internal/repotools/changelog/type.go

```diff
@@ -0,0 +1,151 @@
+package changelog
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// SemVerIncrement describes how a Annotation should affect a module's version.
+type SemVerIncrement int
+
+const (
+	_ SemVerIncrement = iota
```

Comment:
Hmm, the intent of `SemVerIncrement` is to communicate that a specific change type has a specific semantic version increment that should be applied. This mapping is done via `ChangeType#VersionIncrement` method. The odd one that follows into this `0` category right now is `Announcement` change type, since an announcement doesn't imply a specific desired bump behavior on it's own. `Unset` wording seems weird....maybe `Default`? `Default` implies no particular bump but would allow the next version selector to decide. So by default that would essentially be a patch bump.

Reply:
Decided to go with `DefaultBump` for now.